lib/createDIContainer.js

```diff
@@ -132,10 +137,13 @@ async function createDIContainer(options) {
     dataContractLevelDBFile: asValue(options.DATA_CONTRACT_LEVEL_DB_FILE),
     documentMongoDBPrefix: asValue(options.DOCUMENT_MONGODB_DB_PREFIX),
     documentMongoDBUrl: asValue(options.DOCUMENT_MONGODB_URL),
+    chainLock: asValue(undefined),
```

Comment:
What for?

Reply:
I got an error with createDIContainer when this is not existant where it fails to build dep tree.

Reply:
I see. You can just use `asFunction`  for `latestCoreChainLock` and instantiate the class by yourself or remove the constructor at all.